NDcPP_v3_0.adoc

```diff
@@ -1202,7 +1202,7 @@ _The TOE must provide functionality for remote administration. Local administrat
 . _RS-232 terminal._
 . _Peripherals (e.g. keyboard, monitor, mouse)._
 
-_The TOE must provide functionality to configure the access banner for FTA_TAB.1 and the session inactivity time(s) for FTA_SSL.3 and (if included) FTA_SSL_EXT.1._
+_The TOE must provide functionality to configure the access banner for FTA_TAB.1 and the session inactivity time(s) for FTA_SSL.3 and (if included) FTA_SSL_EXT.1, though an access banner is only required for each interactive (human-machine) interface, not for any programmatic interface [application programming interface (API), e.g. REST API]._
```

Comment:
@dundiddat Should the wording be changed to "(human-computer) interface" here?

Reply:
Yes, I should have used "human-computer".  I'll change it.